packages/firestore/test/util/mock_persistence.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as registerIndexedDBShim from 'indexeddbshim';
+import * as fs from 'fs';
+
+// WARNING: The `indexeddbshim` installed via this module should only ever be
+// used during initial development. Do not use this code in your production apps
+// and always validate your changes via `yarn test:browser` (which uses a
+// browser-based IndexedDB implementation) before integrating with Firestore.
```

Comment:
This warning reads a bit strange to me. I'm not sure if the intended audience is Firestore SDK contributors (mentioning "via 'yarn test:browser'"), or Firestore SDK consumers ("production apps"?)...

Reply:
It was meant for both :) But since our customers will likely never see this, I removed that portion.